### PR TITLE
8.0 security and groups revamp project and definitions

### PR DESCRIPTION
**Pull request by gfcapalbo**

### CODE REVIEW

**Comment by hbrunn** [comment]
that looks fine.

In another thread, we were talking about project_user being a too permissive group. I think you should remove the inherit from the collaborators from to this group, not use it at all and attach all the needed permissions to collaborators.

File: trp_project_scrum/security/ir.model.access.csv [hunk 1]
```diff
@@ -14,3 +15,6 @@ collaborators_followers_scrum_product_product,collaborators_followers_scrum_prod
 collaborators_followers_scrum_product_template,collaborators_followers_scrum_product_template,product.model_product_template,collaborators_followers,1,0,0,0
 collaborators_followers_scrum_product_uom,collaborators_followers_scrum_product_uom,product.model_product_uom,collaborators_followers,1,0,0,0
 collaborators_price_type_user,collaborators_price_type_user,product.model_product_price_type,collaborators_followers,1,0,0,0
+access_project_project,project.project,project.model_project_project,collaborators_followers,1,0,0,0
+knowledge.access_document_page_user,document.page,document_page.model_document_page,collaborators_followers,1,1,0,0
+
```
Comment (hbrunn) [inline trp_project_scrum/security/ir.model.access.csv]: delete empty line

**Review by hbrunn — APPROVED** [review]
that's fine, squash your commits for merge

**Comment by gfcapalbo** [comment]
@hbrunn removed empty line and rebased.

**Pull request merged**